src/fixtures/item.ts

```diff
@@ -80,10 +81,119 @@ const mainMedia = {
 	},
 };
 
+const doc = new DocumentFragment();
+
+const el = document.createElement('p');
+
+el.innerText =
+	'Readers of Prospect magazine recently voted him the world’s fourth-best thinker. And right now he is thinking about 3 November, and whether the United States will reject or endorse Donald Trump. No one knows what will happen; not even West, not least because in the US he sees contradictions that even he can’t fully explain.';
+
+doc.appendChild(el);
```

Comment:
just for Marie Kondo reasons, can we put this in a `docFixture()` function?

Reply:
done :)